lib/formatPhone.js

```diff
@@ -0,0 +1,20 @@
+/* global formatPhone:true */
+/* exported formatPhone */
+
+formatPhone = phone => {
+  if (!phone) {
+    return null
+  }
+  const phoneDigits = phone.toString().replace(/\D/g, '')
+  const areaCode = phoneDigits.slice(0, 3)
+  const prefix = phoneDigits.slice(3, 6)
+  const suffix = phoneDigits.slice(6, 10)
+  let formatted = String(areaCode)
+  if (phoneDigits.length > 3) {
+    formatted = `(${areaCode}) ${prefix}`
+  }
+  if (phoneDigits.length > 6) {
+    formatted += `-${suffix}`
+  }
+  return formatted
```

Comment:
If I pass this function a phone number without an area code like `830-5153` the output is `(830) 515`. Maybe we should reverse this and assume that if digits are left off they're left off of the front?